### PR TITLE
Making changes to Bullet Chart  in response to Leslie’s PTNFLY-2117

### PR DESCRIPTION
**Pull request by jennyhaines**
@LHinson 
## **Description of Changes Made**
- Remove the title “Overview”
- We need anchor links like the other pages
- Images are large on the design tab
- Remove title at the top of the page on the design tab
- Remove title “Examples”
- Optional is bold on the design tab
- Don’t need that many bullets
- images names should all be lower case
- image name cleanup

[Jira Story](https://patternfly.atlassian.net/browse/PTNFLY-2117)

### CODE REVIEW

File: pattern-library/data-visualization/donut-chart/design/overview.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Donut Chart
 
-## Overview
 The most common use case for a donut chart are:
 
 Show progress completion as a percentage.
```
Comment (LHinson) [inline pattern-library/data-visualization/donut-chart/design/overview.md]: the use cases should be bulleted to improve readability

File: pattern-library/data-visualization/donut-chart/design/overview.md [hunk 2]
```diff
@@ -9,17 +8,16 @@ Show the relationship of a set of values to a whole (design and specs for this u
...
 Jump to [Utilization](#utilization), [Relationship of a Set of Values to a Whole](#relationship-of-a-set-of-values-to-a-whole) or [Small Donut Chart](#small-donut-chart)
 
-## Examples
 
-### Utilization
+## Utilization
 When using a donut chart to show utilization, the entire donut should add up to 100% and the fill represents a percentage of that. You can show that a threshold has been reach by using red, orange or green colors. Otherwise, use the blue color when thresholds are not present. Consider using the [Utilization Bar Chart](https://patternfly.org/pattern-library/data-visualization/utilization-bar-chart) to show utilization in more limited spaces.
 
-![donut-chart-example-1](img/donut-chart-example-1.png)
+![donut-chart-example-1](img/utilization-donut-chart.png)
```
Comment (LHinson) [inline pattern-library/data-visualization/donut-chart/design/overview.md]: does the resolution on these images = 72? they still look big to me.

File: pattern-library/data-visualization/donut-chart/design/overview.md [hunk 3]
```diff
@@ -9,17 +8,16 @@ Show the relationship of a set of values to a whole (design and specs for this u
...
 
-### Relationship of a Set of Values to a Whole
+## Relationship of a Set of Values to a Whole
 When using a donut chart to show the relationship of a set of values to a whole, the pieces must add up to 100% and there should be no more than six categories. Sections within a donut chart may be hard to compare to each other so you should use a donut chart when you want to compare an individual section to the whole. Consider using a [Bar Chart](https://patternfly.org/pattern-library/data-visualization/bar-chart) if you would like to compare one category to another.
 
-![donut-chart-example-1](img/donut-chart-example-2.png)
+![donut-chart-example-1](img/set-of-values-donut-chart.png)
```
Comment (LHinson) [inline pattern-library/data-visualization/donut-chart/design/overview.md]: does the resolution on these images = 72? they still look big to me.

File: pattern-library/data-visualization/donut-chart/design/overview.md [hunk 4]
```diff
@@ -9,17 +8,16 @@ Show the relationship of a set of values to a whole (design and specs for this u
...
 
-### Small Donut Chart
-![#donut-chart-example-3](img/donut-chart-example-3.png)
+## Small Donut Chart
+![#donut-chart-example-3](img/legend-donut-chart.png)
```
Comment (LHinson) [inline pattern-library/data-visualization/donut-chart/design/overview.md]: does the resolution on these images = 72? they still look big to me.... doesnt' looke like a "small" donut chart :)

File: pattern-library/data-visualization/bullet-chart/design/design.md [hunk 5]
```diff
@@ -1,16 +1,15 @@
 # Bullet Chart
 
-## Standard Bullet Chart Overview
-![Image of standard bullet chart](img/Overview-spec.png)
+![Image of standard bullet chart](img/bullet-chart-callout.png)
 
 1. **Featured Measure Bar:**
   * Use a visually prominent fill color for the featured measure bar. See the [color palette](https://www.patternfly.org/styles/color-palette/) for color recommendations.
```
Comment (LHinson) [inline pattern-library/data-visualization/bullet-chart/design/design.md]: No need to bullet this item. Remove the "enter" and make description come directly after.

File: pattern-library/data-visualization/bullet-chart/design/design.md [hunk 6]
```diff
@@ -1,16 +1,15 @@
 # Bullet Chart
 
-## Standard Bullet Chart Overview
```
Comment (LHinson) [inline pattern-library/data-visualization/bullet-chart/design/design.md]: The bullet chart page description is pretty long, let's add anchor links at the top.

File: pattern-library/data-visualization/donut-chart/design/overview.md (hunk 2 above)
Comment (jennyhaines) [inline pattern-library/data-visualization/donut-chart/design/overview.md]: Yes, the resolution for the images is 72. I had downsized them from 100% -> 85%, but will make even smaller.

**Pull request closed without merging**